frontend/src/v4/routes/components/messagesList/components/message/components/markdownMessage/markdownMessage.component.tsx

```diff
@@ -16,20 +16,15 @@
  */
 
 import ReactMarkdown from 'react-markdown';
-import breaks from 'remark-breaks';
 
-import { renderers } from './markdownMessage.renderers';
+import { components } from './markdownMessage.renderers';
 
 type MarkdownMessageProps = {
 	className?: string;
 	children: any;
 };
-
 export const MarkdownMessage = ({ className, children }: MarkdownMessageProps) => (
-	<ReactMarkdown
-		source={children}
-		renderers={renderers}
-		plugins={[breaks]}
-		className={className}
-	/>
+	<ReactMarkdown components={components} className={className}>
```

Comment:
Quote and reply (marked below) don't seem to be working.

This example is in production

<img width="335" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/be6a08e8-4673-4382-86d3-4524fc7bcede">